typescript/sdk/src/gas/HyperlaneIgpDeployer.ts

```diff
@@ -40,17 +39,19 @@ export class HyperlaneIgpDeployer extends HyperlaneDeployer<
     chain: ChainName,
     proxyAdmin: ProxyAdmin,
     storageGasOracle: StorageGasOracle,
-    deployOpts?: DeployOptions,
   ): Promise<InterchainGasPaymaster> {
-    const owner = this.configMap[chain].owner;
     const beneficiary = this.configMap[chain].beneficiary;
+    const owner = await this.multiProvider.getSignerAddress(chain);
+    // The IGP initializes itself in the constructor, so we do not
+    // pass initialization args.
+    // Wait scratch that, we want to initialize the proxied version but not the
+    // un-proxied version.
```

Comment:
whats the decision here?

Reply:
Oops, this shoudl be removed. The IGP no longer inits itself in the constructor